### PR TITLE
Add LT3471

### PR DESCRIPTION
**Pull request by evanshultz**
https://www.analog.com/media/en/technical-documentation/data-sheets/3471fb.pdf
image

Notes:
- VIN to SW* pins are 400mil apart to fit a small inductor nicely
- VREF is Power Output since it can source 1mA and is intended for that purpose

Also:
- Fix LT3472 footprint since it's shared
- Change LT3472 pin 3 style to 'Line'
- Make L/R LT3472 pins 100mil long and widen symbol to compensate (you can see this below that no pins changed positions so it's a safe fix)

Footprint PR: https://github.com/KiCad/kicad-footprints/pull/1648

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the symbol(s) you are contributing
- [ ] An example screenshot image is very helpful
- [ ] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

### CODE REVIEW

**Comment by Misca1234** [comment]
reopen to triger travis

File: Regulator_Switching.lib [hunk 1]
```diff
@@ -2946,28 +2946,54 @@ X NC 9 -300 0 100 R 50 50 1 1 N N
...
 $ENDFPLIST
 DRAW
-S -450 300 450 -400 0 1 10 f
+S -500 300 500 -400 0 1 10 f
 X SWP 1 -100 400 100 D 50 50 1 1 W
-X VPOS 10 -600 200 150 R 50 50 1 1 O
+X VPOS 10 -600 200 100 R 50 50 1 1 O
 X GND 11 0 -500 100 U 50 50 1 1 W
 X VIN 2 0 400 100 D 50 50 1 1 W
-X SHDN 3 -600 -300 150 R 50 50 1 1 I I
```
Comment (diegoherranz) [inline Regulator_Switching.lib]: What is that extra `I` that was there? Was it simply wrong?

Comment (diegoherranz) [inline Regulator_Switching.lib]: Given you've removed the inverted line style, can you put a bar on top of SHDN?

Comment (diegoherranz) [reply Regulator_Switching.lib]: Nevermind, I've just noticed that was the "inverted" line style :)

**Comment by diegoherranz** [comment]
Except for that comment above, it all looks good. I've checked:

- Pinout
- Pin types
- Pin arrangement: looks sensible
- Description
- Datasheet link

Regarding the fixes for LT3472:

- The new footprint is better since it matches the datasheet exactly.
- Rectangle size changed and pins made shorter. All the connections should still be at the same place.

So as soon as the `SHDN` pin is renamed to `~SHDN`, it's ready to be merged.

Thanks!

**Comment by evanshultz** [comment]
@diegoherranz 
Good catch. I've added the overbar. Thanks!

**Comment by diegoherranz** [comment]
Thanks, @evanshultz. Merging.

**Pull request merged**